go.mod

```diff
@@ -7,6 +7,7 @@ replace (
 	github.com/hashicorp/consul/envoyextensions => ./envoyextensions
 	github.com/hashicorp/consul/proto-public => ./proto-public
 	github.com/hashicorp/consul/sdk => ./sdk
+	github.com/hashicorp/consul/testingconsul => ../testingconsul
```

Comment:
This doesn't seem correct

Reply:
Oh, I don't think that needs to be in there anyway.